src/main/java/bio/terra/service/resourcemanagement/google/GoogleProjectService.java

```diff
@@ -479,16 +477,18 @@ private static void enableFirestore(
   static String extractOperationIdFromName(final String appId, final String opName) {
     // The format returns is apps/{appId}/operations/{useful id} so we need to extract it
     // Add a check in case they ever change the format
-    final String uuidRegex =
-        "\\p{XDigit}{8}-\\p{XDigit}{4}-\\p{XDigit}{4}-\\p{XDigit}{4}-\\p{XDigit}{12}";
-    final Pattern pattern =
-        Pattern.compile(String.format("^apps/%s/operations/(%s)", appId, uuidRegex));
-    final Matcher matcher = pattern.matcher(opName);
-    if (!matcher.find()) {
+    String expectedPrefix = String.format("apps/%s/operations/", appId);
+    if (opName.startsWith(expectedPrefix)) {
+      var split = opName.split("/");
+      if (split.length == 4) {
+        return split[3];
+      }
       throw new AppengineException(
-          String.format("Operation Name does not look as expected: %s", opName));
+          String.format("Operation Name %s expected to have exactly 4 elements", opName));
     }
```

Comment:
Makes sense to me -- We can't guarantee anything about the string except for the prefix.